testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/page/SigningInPage.java

```diff
@@ -192,6 +192,15 @@ public String getUserLabel() {
             return getTextFromItem(LABEL);
         }
 
+        public boolean hasCreatedAt() {
+            boolean result = false;
+            try {
+                result = getItemElement(CREATED_AT) != null;
```

Comment:
`getItemElement` can never return null. It will either return the element or throw an exception.